src/index.jsx

```diff
@@ -17,7 +18,10 @@ const AppWrapper = () => (
     <ConnectedRouter history={history}>
       <CookiesProvider>
         <Router>
-          <App />
+          <Switch>
```

Comment:
As I mentioned in another comment, I think it makes sense to move the `Header` and `Footer` into this component under `Router`. Right now, you're using the `Footer` within the `LoginPage` component. The `LoginPage` component should only know about things related to the login page (i.e., only the form).

If you move the `Header` and `Footer` into `AppWrapper` you'd get the header and footer for free on the login page.